File: DHTweetAppAPI-masterWithMongoDB/src/main/java/com/tweetapp/model/Tweets.java
package com.tweetapp.model;

import java.time.LocalDateTime;
import java.util.List;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
@Document(collection = "Tweets")
public class Tweets {

	@Id
	private String tweetId;
	private String tweetBody;
	private String userName;
	private LocalDateTime datetime;
	private List<String> likedBy;
	private List<TweetsReply> replies;
}

File: DHTweetAppUI-master/src/app/model/tweets.ts
import { TweetsReply } from "./tweetsReply";

export class Tweets {
    tweetId?: string
    userName?: string
    tweetBody?: string
    datetime?: Date
    likedBy?: string[]
    replies?: TweetsReply[]

    // constructor(
    //     datetime?: Date,
    // ) {
    //     this.datetime = datetime || new Date();
    // }
}